hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -153,10 +157,13 @@ private void scan() {
             long now = Clock.currentTimeMillis();
             int backupTimeouts = 0;
             int invocationTimeouts = 0;
+            int invocationCount = 0;
```

Comment:
Yes :) Maximum number of concurrent invocations is bound to the size of the CHM.... and therefor an int.